python/hidet/option.py

```diff
@@ -701,12 +709,28 @@ def debug_cache_tuning(enabled: bool = True):
     OptionContext.current().set_option('debug_cache_tuning', enabled)
 
 
+def debug_enable_var_id(enable: bool = True):
+    """
+    Whether to enable var id in the IR.
+
+    When this option is enabled, each variable (i.e., hidet.ir.Var) will have a unique id.
+    Otherwise, each variable's ID will be 0.
+
+    Parameters
+    ----------
+    enable: bool
+        Whether to enable var id in the IR.
+    """
+    OptionContext.current().set_option('debug_enable_var_id', enable)
+
+
 def debug_show_var_id(enable: bool = True):
     """
     Whether to show the var id in the IR.
 
     When this option is enabled, the IR will show the var id with the format `var@id`, like `x@1` and `d_1@1732`.
     Variable (i.e., hidet.ir.Var) a and b is the same var if and only if `a is b` evaluates to True in Python).
+    Hint: all variable ids will always be 0 unless the debug_enable_var_id option is set to True.',
```

Comment:
Maybe check whether the current `debug_enable_var_id` is True or not when we turn on `debug_show_var_id`. If `debug_enable_var_id` is off, we can use
```python
warnings.warn("Please use `hidet.option.debug_enable_var_id()` to enable the id first")
```
to ask the user to turn on the id tracking.
Alternatively, we can silently turn it on.